frontend/src/core/project/components/ProjectMenu.test.js

```diff
@@ -58,7 +60,7 @@ describe('<ProjectMenu>', () => {
         const wrapper = createShallowProjectMenu({ project: ALL_PROJECTS });
 
         expect(wrapper.find('.project-menu .selector')).toHaveLength(1);
-        expect(wrapper.find('.project-menu .selector span:first-child').text()).toEqual('All Projects');
+        expect(findLocalizedById(wrapper, 'resource-ResourceMenu--all-projects')).toHaveLength(1);
```

Comment:
Wouldn't that work as well (and be more elegant)?
```js
expect(wrapper.find('#resource-ResourceMenu--all-projects')).toHaveLength(1);
```

Reply:
sure it will work.

Reply:
Nice!